app/src/main/res/layout/activity_main_page.xml

```diff
@@ -131,7 +131,7 @@
                             android:layout_height="@dimen/image_button_home_page"
                             android:layout_gravity="center_horizontal"
                             android:layout_marginTop="@dimen/margin_image_inside_layout"
-                            android:background="@drawable/hospital"
+                            android:background="@drawable/hospital2"
```

Comment:
They are actually completely separate buttons, the problem is that the alpha attribute of the image gets overwritten by clicking on it, and since both buttons use the same image resource, it appears as it was also clicked in the main.

Reply:
My idea was just to reset the alpha variable inside the onResume method of the main activity, thus when the user goes back to the main page they appear unselected, this would avoid having to add 3 more similar images to our project, but it's not a big deal so as you want

Reply:
Yes I got it ! But the thing is that if you deselect the user's choices, he/she will have to reselect them each time he enters the map which is not practical. We can still merge them into 1 single button in the main page in a future task to avoid duplication of images and functionalities.